### PR TITLE
Allow a super admin to check the register before adding an organisation

### PR DESCRIPTION
**Pull request by georgeschena**
Screenshot 2019-04-10 at 13 31 09

### CODE REVIEW

File: app/controllers/admin/custom_organisations_controller.rb [hunk 1]
```diff
@@ -12,6 +14,7 @@ def create
 
     if @custom_organisation.save
       flash.now[:notice] = "Custom organisation has been successfully added"
+      set_organisations_from_register
```
Comment (emileswarts) [inline app/controllers/admin/custom_organisations_controller.rb]: Isn't this run in the before action already?

Comment (georgeschena) [reply app/controllers/admin/custom_organisations_controller.rb]: Hahah let me explain

File: app/controllers/admin/custom_organisations_controller.rb [hunk 2]
```diff
@@ -1,6 +1,8 @@
 class Admin::CustomOrganisationsController < AdminController
   helper_method :sort_column, :sort_direction
 
+  before_action :set_organisations_from_register, only: %i[create index]
```
Comment (emileswarts) [inline app/controllers/admin/custom_organisations_controller.rb]: Where is this instance variable being used for the create action?

File: app/controllers/admin/custom_organisations_controller.rb (hunk 1 above)
Comment (georgeschena) [reply app/controllers/admin/custom_organisations_controller.rb]: Easier face to face

Comment (georgeschena) [reply app/controllers/admin/custom_organisations_controller.rb]: As its bit odd

File: app/views/admin/custom_organisations/index.html.erb [hunk 3]
```diff
@@ -37,3 +39,18 @@
...
+  accessibleAutocomplete.enhanceSelectElement({
+    selectElement: document.querySelector('#organisations_register'),
+    required: true,
+    displayMenu: 'overlay',
+    defaultValue: '',
+    showNoOptionsFound: true,
+    showAllValues: true,
+    confirmOnBlur: true
+  })
+</script>
```
Comment (emileswarts) [inline app/views/admin/custom_organisations/index.html.erb]: Could we pull this out into a partial so it's not duplicated?

**Pull request merged**